a.py

```diff
@@ -0,0 +1,23 @@
+import torch
+import torch._dynamo.config
+
+
+
+torch._inductor.config.aoti_wrapper = True
```

Comment:
This is ABSOLUTELY not what the final API should be, this is enough jank to get the test script to run